### ISSUE TITLE
No TNT change compared to vanilla

### ISSUE THREAD

**Issue by JenkoRun** [MLG-Fortress/ExtraHardMode#198]
I and a few players have noticed that TNT is not behaving as we expected, I've tried loading EHM on a private server with no other plugins and increasing the explosion power for above and below border up to 50, yet TNT does not seem to increase its explosion size at all.

MC Server version: git-paper-221

EHM Version: 3.14.0-e490375

I've tried enabling DEBUG but nothing came through in the console related to TNT or even block changes. Please advise.

**Comment by Hastur042** [comment]
I'm seeing the same thing.  Disabled other plugins and changed the tnt explosion power but it does not increase the explosion damage at all.   Seems to just be vanilla behavior.

I'm using the latest spigot 1.14.4
ehm: 3.14.0-e490375

**Comment by RoboMWM** [comment]
Hmm, wonder if this has been changed/deprecated in the spigot API.

**Comment by Hastur042** [comment]
Not sure if it is important... but I noticed in the startup log that max
TNT dmg is showing as 100... which is set in the spigot.yml.  No idea if
that is overriding EHM.

**Comment by RoboMWM** [comment]
It's using `world#createExplosion(double x, double y, double z, float power, boolean setFire, boolean breakBlocks)` so perhaps spigot is no longer using that value. Maybe it has a cap. Idk.

**Comment by Hastur042** [comment]
I noticed my spigot version was behind 3 versions, updated to current
again, TNT damage still regular.

Switched the server to use craftbukkit-1.14.4-R01-snapshot.jar ...  TNT
damage is regular also in EHM.

Any recommendations on how this can be fixed?  The TNT mechanic in EHM is
pretty important with the weakened iron tools.

Many thanks in advance!

**Comment by RoboMWM** [comment]
The multiple explosions should provide extra power. I'll see if I can write something to test plugin-generated explosions.

**Comment by IR0NSIGHT** [comment]
i noticed it doesnt generate multiple explosions as well. 
Not above nor below the border. TNT seems to be rewind to vanilla completely, except the "shoot blocks away like debris" thing.

**Comment by RoboMWM** [comment]
Multiple explosions aren't created? Please test with older versions. It's not easy to see the multiple explosions as they all happen simultaneously.

**Comment by IR0NSIGHT** [comment]
There seems to be no difference in the crater size, no matter if multiexplosions is true or false in the config.
Check my test here:
https://imgur.com/ltSVw0Q

**Comment by IR0NSIGHT** [comment]
Here visual of testing in EHM 3.14.2, spigot 1.14.4 with tnt and charged creepers:
https://imgur.com/HoBAdSR

**Comment by IR0NSIGHT** [comment]
Sorry if i keep spamming here.
I found this plugin which lets you configure TNT.
If you set tnt radius to 50 or 70 it works like EHM tnt used to.
Maybe RoboMWM can have a look to figure out the explosion functions as well.

https://www.spigotmc.org/resources/ultimatetnt-the-ultimate-tnt-plugin.49388/

**Comment by RoboMWM** [comment]
Need comparisons to older versions

**Comment by LoonyLadle** [comment]
Can confirm that TNT behavior is back to vanilla. Brand new server, no other plugins.

**Comment by RoboMWM** [comment]
> Need comparisons to older versions

**Comment by LoonyLadle** [comment]
What sort of comparison are you looking for? It's been years since I last ran a Minecraft server.

**Comment by RoboMWM** [comment]
> i noticed it doesnt generate multiple explosions as well.

Is also puzzling to me - was this the case for you as well?

**Comment by LoonyLadle** [comment]
"TNT behavior is back to vanilla" means "TNT behavior is not EHM's TNT behavior" which means, yes, no multiple explosions, no customizable size, just the standard no-plugin behavior.

**Comment by RoboMWM** [comment]
> compiling Spigot for 1.14.4 took an hour. 

Try using Paper? https://papermc.io/legacy

**Comment by RoboMWM** [comment]
I checked that ultimatetnt plugin, and it's manually performing logic to remove blocks within a specified radius.

I made a test plugin and can confirm that world#createExplosion works on 1.16.1.

I then tested with EHM and changed the relevant config options (...TnT.Below Border.Explosion Power and the Above Border node as well) and observed similar behavior to my test plugin.

Since no configurations have been provided here I'm going to assume either you guys made an error in the configuration (I did initially, I changed explosion power for creeper instead of TNT - though explosion power for creepers did also work as intended) or there was something wrong with spigot's API in the prior version for that method.
